src/assessment/templates/assessment/assigned_work_list.html

```diff
@@ -0,0 +1,8 @@
+<h1>Assigned work</h1>
```

Comment:
@avishkar-sapkal remove the template folder from assessment and move under `src.templates.assessment`. Use the right structure for templates